scripts/deploy.js

```diff
@@ -0,0 +1,13 @@
+const exec = require('child_process').exec;
+
+exec('npm publish', (error, stdout, stderr) => {
```

Comment:
I was thinking - does this actually do anything than just running npm publish directly? Also, should we be yarn publishing instead?